docs/DeveloperGuide.md

```diff
@@ -85,11 +85,11 @@ The `UI` component,
 
 ### Logic component
 
-**API** : [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
+**API** : [`InternshipLogic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
 
-Here's a (partial) class diagram of the `Logic` component:
+Here's a (partial) class diagram of the `InternshipLogic` component:
 
-<img src="images/LogicClassDiagram.png" width="550"/>
+<img src="images/InternshipLogicClassDiagram.png" width="550"/>
 
 The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
```

Comment:
Change logic to InternshipLogic

Reply:
In my latest commit, I changed all references of "Logic" to "InternshipLogic"